shell/ci/auth/github_packages.sh

```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env bash
 # Sets up Github Packages for a variety of languages.
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+SHELL_DIR="$DIR/../.."
+LIB_DIR="${DIR}/../../lib"
 
-tokenFile="$HOME/.outreach/github.token"
+# shellcheck source=../../lib/bootstrap.sh
+source "${LIB_DIR}/bootstrap.sh"
 
-if [[ ! -e $tokenFile ]]; then
-  echo "Skipped: Github Auth not setup"
-fi
-
-# Allow overriding the token, e.g. for uploading packages
-# you need to use a PAT vs a Github App token.
-GH_TOKEN="$(cat "$tokenFile")"
-if [[ -n $GITHUB_PACKAGES_TOKEN ]]; then
-  GH_TOKEN=$GITHUB_PACKAGES_TOKEN
-fi
+GH_TOKEN=$("$SHELL_DIR/gobin.sh" \
+  "github.com/getoutreach/ci/cmd/ghaccesstoken@$(get_tool_version "getoutreach/ci")" \
+  --skip-update token --env-prefix "GHACCESSTOKEN_PAT")
```

Comment:
This is set in the CircleCI context `ghaccesstoken`. This allows us to also pool these in the future if needed.